build-tools/packages/build-cli/docs/generate.md

```diff
@@ -114,18 +114,17 @@ Generates type tests based on the individual package settings in package.json.
 ```
 USAGE
   $ flub generate typetests [-d <value> | --packages | -g client|server|azure|build-tools] [--prepare | --generate]
```

Comment:
Could you clarify --packages: "Run on all independent packages in the repo." ?

Reply:
It's a shortcut to run generation on all independent packages, like build-common, common-utils, etc. The alternative is to use the `--dir` option and pass the path to the package, but this is more convenient when you're updating them all. Clarified in the flag docs.